### PR TITLE
Add adb commands

### PR DESCRIPTION
**Pull request by lagelalegal**

### CODE REVIEW

File: pages/common/adb.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Android Debug Bridge
+
+> Android Debug Bridge (adb) is a versatile command line tool
```
Comment (igorshubovych) [inline pages/common/adb.md]: You already said that it is Android Debug Bridge. No need to repeat it here.
I would rather say about its features.
```md
> Communicate with an emulator instance or connected Android-powered devices
```

**Comment by igorshubovych** [comment]
:+1:

**Comment by lagelalegal** [comment]
thanks, good idea

File: pages/common/adb.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Android Debug Bridge
+
+> Communicate with an emulator instance or connected Android-powered devices
```
Comment (waldyrious) [inline pages/common/adb.md]: Suggestions: "emulator" --> "Android emulator", "Android-powered devices" --> "Android devices".

**Comment by waldyrious** [comment]
Please use the infinitive tense in the command descriptions ("Check" rather than "Checks", etc.)

**Comment by lagelalegal** [comment]
@waldyrious  I'm sorry, My English is not so good. I'll try my best to improve it.

**Comment by felixonmars** [comment]
:+1:

**Comment by waldyrious** [comment]
@lagelalegal those weren't English mistakes, just writing style suggestions :)

**Pull request merged**